openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -192,9 +193,25 @@ void WriteAssessmentFile(std::string const fileName, pugi::xml_document const &
   }
 
   pugi::xml_document result;
+  auto segments = result.append_child("Segments");
+  auto const dict = doc.select_node(".//Dictionary").node();
+
+  // Copy namespaces from <Dictionary> to <Segments>
+  for (auto const attr : dict.attributes())
+  {
+    if (strncmp("xmlns", attr.name(), 5) != 0)
```

Comment:
I suggest you to move "xmlns" out to a separate variable and use its .size() method instead of the literal "5".

Reply:
+1

Reply:
Done.